cassandra/README.md

```diff
@@ -420,433 +371,16 @@ $ grace=$(kubectl get po cassandra-0 --template '{{.spec.terminationGracePeriodS
-UN  10.244.0.5  74.09 KB   256     100.0%            86feda0f-f070-4a5b-bda1-2eeb0ad08b77  rack1
-UN  10.244.4.2  32.45 KB   256     100.0%            0b1be71a-6ffb-4895-ac3e-b9791299c141  rack1
-UN  10.244.3.3  51.28 KB   256     100.0%            dafe3154-1d67-42e1-ac1d-78e7e80dce2b  rack1
-
-```
-
-**Note**: This example had you delete the cassandra Replication Controller before
-you created the DaemonSet.  This is because – to keep this example simple – the
-RC and the DaemonSet are using the same `app=cassandra` label (so that their pods map to the
-service we created, and so that the SeedProvider can identify them).
-
-If we didn't delete the RC first, the two resources would conflict with
-respect to how many pods they wanted to have running. If we wanted, we could support running
-both together by using additional labels and selectors.
-
-## Step 9: Resource Cleanup
-
-When you are ready to take down your resources, do the following:
-
-```console
-
-$ kubectl delete service -l app=cassandra
-$ kubectl delete daemonset cassandra
-
+  && kubectl delete pvc -l app=cassandra \
+  && kubectl delete svc cassandra
 ```
 
-### Custom Seed Provider
-
-A custom [`SeedProvider`](https://svn.apache.org/repos/asf/cassandra/trunk/src/java/org/apache/cassandra/locator/SeedProvider.java)
-is included for running Cassandra on top of Kubernetes.  Only when you deploy Cassandra
-via a replication control or a daemonset, you will need to use the custom seed provider.
-In Cassandra, a `SeedProvider` bootstraps the gossip protocol that Cassandra uses to find other
-Cassandra nodes. Seed addresses are hosts deemed as contact points. Cassandra
-instances use the seed list to find each other and learn the topology of the
-ring. The [`KubernetesSeedProvider`](java/src/main/java/io/k8s/cassandra/KubernetesSeedProvider.java)
-discovers Cassandra seeds IP addresses via the Kubernetes API, those Cassandra
-instances are defined within the Cassandra Service.
-
-Refer to the custom seed provider [README](java/README.md) for further
-`KubernetesSeedProvider` configurations. For this example you should not need
-to customize the Seed Provider configurations.
-
-See the [image](image/) directory of this example for specifics on
-how the container docker image was built and what it contains.
-
-You may also note that we are setting some Cassandra parameters (`MAX_HEAP_SIZE`
-and `HEAP_NEWSIZE`), and adding information about the
-[namespace](../../../docs/user-guide/namespaces.md).
-We also tell Kubernetes that the container exposes
-both the `CQL` and `Thrift` API ports.  Finally, we tell the cluster
-manager that we need 0.1 cpu (0.1 core).
+[1]: https://kubernetes.io/docs/concepts/workloads/pods/pod/
+[2]: https://kubernetes.io/docs/concepts/services-networking/service/
+[3]: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/
+[Cassandra]: [http://cassandra.apache.org/]
+[container registry]: [https://quay.io/repository/vorstella/cassandra-k8s]
```

Comment:
I can help you update the `gcr.io/google-samples/cassandra` image if necessary, if you check in the Dockerfile.

Currently with this patch, we won't have the Dockerfile and you'll be the only one who has write access to the image.